src/Cns.ts

```diff
@@ -130,6 +132,29 @@ export default class Cns extends EthereumNamingService {
     return await this.getAllRecords(resolverContract, tokenId);
   }
 
+  async twitter(domain: string): Promise<string> {
+    const verificationAddress = '0x9457f96845DA5bF00408AC1837F3eF3D35ce4563';
+    const twitterHandle = await this.record(domain, 'social.twitter.username');
+    const owner = await this.owner(domain);
+    const validationSignature = await this.record(
+      domain,
+      'validation.social.twitter.username',
+    );
+    const message = [domain, owner, 'validation.twitter.name', twitterHandle]
```

Comment:
@DeRain shouldn't it be `validation.social.twitter.name`?

Reply:
@bogdan @sudoryan 
`social.twitter.username` stores the twitter username
`validation.social.twitter.username` - strores validation signature.

For the message construction, this should be `social.twitter.username`